cmd/influx/template.go

```diff
@@ -344,6 +345,10 @@ func (b *cmdPkgBuilder) cmdExport() *cobra.Command {
 		# export a stack with resources not associated with the stack
 		influx export --stack-id $STACK_ID --buckets $BUCKET_ID
 
+		# export a stack with resources not associated with the stack
+		# and update the stack resources with all exported resources 
+		influx export --update-stack --stack-id $STACK_ID --buckets $BUCKET_ID
```

Comment:
@sanderson this command has been extended with the new `--update-stack` functionality. I'm still struggling with what to call this flag. I'm not sure about `--append`, I'd like it to be more explicit that the append is to the stack's internally managed resoruces... I'm not sure how to phrase this 😢

Reply:
What about `--include`? "Include resources in the exported template"?